xpdan/data_reduction.py

```diff
@@ -149,7 +151,7 @@ def _file_name(self, event, event_timestamp, ind):
 # init
 xpd_data_proc = DataReduction()
 ai = AzimuthalIntegrator()
-
+geo = Geometry()
```

Comment:
I think you can just use the AzimuthalIntegrator for all the Geometry stuff. The integrator inherits from the geometry so it should have all the same methods